.gitignore

```diff
@@ -31,6 +31,7 @@ __pycache__
 
 # PyTest
 .pytest_cache
+test_keys/
```

Comment:
```suggestion
# Ignore key materials generated by examples or tests
test_keys/
```

Reply:
This is so someone looking at the root project's .gitignore has some sense of where this comes from